src/Apps/Order/Components/CreditCardPicker.tsx

```diff
@@ -339,6 +339,7 @@ export class CreditCardPicker extends React.Component<
             <Collapse open={this.needsAddress()}>
```

Comment:
The `CreditCardPicker` needs to be converted to a functional component. I'm going to create a separate ticket for this.